.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "node-services",
+    "prettier"
+  ],
+  "plugins": [
+    "prettier"
```

Comment:
I've found it useful as JSON files often exist side-by-side with JavaScript. I'll try to make a PR for this and see if I can convince you.